### DISCUSSION TITLE
How to add an on click event on mentioned item?

### DISCUSSION THREAD

**Discussion by debamishra14** [ueberdosis/tiptap#851] [General Questions]
I am using tiptap for quite some time and I have a requirement where I have to add an onclick event to the mentioned item so is there any way we can do this ?

@philippkuehn @hanspagel can you guys help me here. any suggestion is highly appreciated

**Comment by hanspagel** [comment]
Hi! Thanks for using tiptap! If you’re code is based on our example you can change the lines highlighted here:
https://github.com/ueberdosis/tiptap/blob/main/examples/Components/Routes/Suggestions/index.vue#L18-L26

If you want to get rid of the default @çlick, you can just point that to another function. Or if you want to keep both, make two divs out of the single div and add to different click handlers there.

Hope that helps as a starting point!

**Reply by debamishra14** [reply]
@hanspagel thanks for your answer, but that's, not my requirement, My requirement is, suppose I have mentioned you, then If I click on your name shown after '@' then a function should get called.

**Reply by hanspagel** [reply]
Ah, got it! Then you have to copy the mention node and write a custom one. You have to change how it’s rendered, highlighted here:
https://github.com/ueberdosis/tiptap/blob/main/packages/tiptap-extensions/src/nodes/Mention.js#L33-L50

It would probably help to use Vue to render that. Have a look at the TodoItem extension:
https://github.com/ueberdosis/tiptap/blob/main/packages/tiptap-extensions/src/nodes/TodoItem.js#L16-L33

**Reply by Billie-Angelov** [reply]
@hanspagel the links are redirecting to 404. Is the example still available somewhere? I need this really bad ! Thanks in advance

**Reply by command-tab** [reply]
Perhaps https://github.com/ueberdosis/tiptap/blob/main/packages/extension-mention/src/mention.ts and the Vue extension docs here will help https://www.tiptap.dev/guide/node-views/vue

**Comment by lleeeuuuuanchik** [comment]
this is how i resolved this problem: 
Mention.configure({
      renderLabel({ options, node }) {
          return `${node.attrs.label}`
      },
      HTMLAttributes: {
          class: 'mention',
          onclick: 'window.open(`/team/user/${event.target.getAttribute("data-id")}`, `_blank`)',
      },
      suggestion,
  }),

i just added attribute onclick to mention configuration and it works
